static/app/components/core/slider/index.tsx

```diff
@@ -128,3 +128,82 @@ function SliderStyles(p: {theme: Theme}) {
     }
   `;
 }
+
+function ChonkSliderStyles(p: {theme: DO_NOT_USE_ChonkTheme}) {
+  return css`
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -webkit-appearance: none;
+    appearance: none;
+    width: 100%;
+    background: transparent;
+    border-radius: ${p.theme.radius.lg};
+    transition: box-shadow 0.1s;
+    box-shadow:
+      0 0 0 8px transparent,
+      0 0 0 10px transparent;
+
+    &:focus {
+      outline: none;
+      box-shadow:
+        0 0 0 8px ${p.theme.background},
+        0 0 0 10px ${p.theme.focusBorder};
+    }
+
+    &[disabled] {
+      cursor: not-allowed;
+      opacity: 0.6;
+
+      &::-webkit-slider-runnable-track {
+        cursor: not-allowed;
+      }
+
+      &::-webkit-slider-thumb {
+        cursor: not-allowed;
+      }
+    }
+
+    /* Chrome styling */
+    &::-webkit-slider-runnable-track {
```

Comment:
Can we combine chrome + ff since the content is the same?

```suggestion
    &::-webkit-slider-runnable-track,
    &::-moz-range-track {
```

Reply:
I will try this again tomorrow. I made this exact change on the old styling and it broke everything 😂

Reply:
@natemoo-re all the styles are broken if I do this :D

Reply:
oh super weird, would have definitely expected that to work! no worries tho, maybe add a comment to that effect?